colossalai/inference/struct.py

```diff
@@ -74,6 +75,7 @@ class Sequence:
     eos_token_id: int
     pad_token_id: int
     max_output_len: int = 256
+    ignore_eos_token_id: bool = False
```

Comment:
You could add a NOTE here to indicate this is a temporary solution. It's better to move the logic to turn on or off the flag in sampling module in future.